### PR TITLE
Create game and author class

### PR DESCRIPTION
**Pull request by Ouail-01**
> ### In this pull request I added the games and author classes. 
The changes made include:

- [X] Create a Game class in a separate .rb file.
- [X] Create an Author class with an association to the Item class (in a separate .rb file).
- [X] All Game class properties visible in the diagram should be defined and set up in the constructor method.
- [X] All Author class properties visible in the diagram should be defined and set up in the constructor method.

Screenshot 2023-07-05 at 07 30 57
Screenshot 2023-07-05 at 07 31 02

### CODE REVIEW

File: classes/games.rb [hunk 1]
```diff
@@ -1 +1,12 @@
...
+
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+end
```
Comment (mahabubx7) [inline classes/games.rb:12]: I think you forgot the `can_be_archived` method to implement it.
Screenshot 2023-07-05 130728

**Review by mahabubx7 — CHANGES_REQUESTED** [review]
Hi Team, 

### 🏃 You are almost there! 🏎️   
You did a great job. But I think you have missed something to make this complete work 🏃‍♂️.
Please take a look in the mentioned comments and made the required changes🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

gif

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/games.rb (hunk 1 above)
Comment (Ouail-01) [reply classes/games.rb:12]: Hello @mahabubx7,

Thank you for your review and feedback. I appreciate your input. I wanted to let you know that I have taken note of your suggestions, and I will be implementing the methods as discussed in the next pull request, which will be labelled, "Implement Methods, #20."

Comment (mahabubx7) [reply classes/games.rb:12]: oh... my bad!

**Review by mahabubx7 — APPROVED** [review]
# Approved ✅

**Pull request merged**